grammar.js

```diff
@@ -138,17 +138,16 @@ module.exports = grammar({
 
     assert_statement: $ => seq(
       'assert',
-      $._expression,
-      repeat(seq(',', $._expression))
+      $.expression_list
     ),
 
-    expression_statement: $ => choice(
+    expression_statement: $ => prec(PREC.expression_statement, choice(
```

Comment:
I've given `expression_statement` a slightly higher precedent than `expression_list`, because `expression_statement` subsumes `expression_list`. This prevents an ambiguity between `expression_list` and `expression_statement`.